infra/src/test/kotlin/fi/fta/geoviite/infra/ui/pagemodel/map/MapNavigationPanel.kt

```diff
@@ -81,6 +84,16 @@ class MapNavigationPanel {
         }
     }
 
+    fun waitForLocationTrackNamesTo(namePredicate: (names: List<String>) -> Boolean) {
+        logger.info("Wait for location track names")
+        WebDriverWait(browser(), Duration.ofSeconds(5))
+            .until { _ ->
+                namePredicate(DynamicList(By.xpath("//ol[@qa-id='location-tracks-list']")).listElements()
```

Comment:
Jos tulee jotain hyviä ajatuksia niin näihin seleniumin elementtihakuihin voisi myös rakentaa jonkun kivan patternin jolla olisi helpompi käyttää. Xpathin voisi esim rakentaa builder-patternilla `ol().attribute("qa-id" to "location-tracks-list")`, tai ihan vain staattisella funkkarilla jossa olisi vararg attribuutit tms. Ei siis tässä tarvitse korjailla mutta jos tulee hyviä ajatuksia niin voi rempata jossain kohtaa.